casa/OS/Path.cc

```diff
@@ -395,23 +395,12 @@ String Path::makeAbsoluteName (const String& inString) const
 	return inString;
     }
     // Otherwise we have a relative pathname.
-    // Remove a possible leading . or ./
+    // Remove a possible leading ./
     String workString (inString);
-    if (workString.length() > 0) {
-	if (workString[0] == '.') {
-	    Int from(1);
-	    if (workString.length() > 1){
-               switch(workString[1]) {
-                case '/' :
-		   from = 2;
-                   break;
-                case '.' :
-                   from = 0;
-                   break;
-               }
-	    } 
-	    workString = workString.from (from);
-	}
+    if (workString == ".") {
+        workString = "";
+    } else if (workString.before(2) == "./") {
```

Comment:
```suggestion
    } else if (workString.startsWith("./")) {
```
For readability I would change this, especially since `std::string::starts_with` exists too.